src/main/docker/cypress/application.yml

```diff
@@ -39,7 +39,8 @@ artemis:
   repo-download-clone-path: repos-download
   file-upload-path: ./uploads
   submission-export-path: exports
-  encryption-password: cypressEncryptionPassword
+  encryption-password: cypressEncryptionPassword # LEGACY: arbitrary password for encrypting database values
+  bcrypt-salt-rounds: 4    # We don't need secure passwords for testing. Lower rounds will speed up tests. 4 ist the lowest
```

Comment:
Please document the range in the other configuration files & the documentation. (Maybe I didn't see it, if it is already present :) )

Reply:
You are supposed to use the benchmarking tool to determine the best configuration for your system. That is documented. I'd argue that is enough. What do you think?

Reply:
Fine for me :) I just thought that this lower bound 4 is not documented at other places.

Reply:
Yes, this is correct. My reasoning was: It's relevant to specify this for testing to understand why it's `4` but for the production configuration it's not relevant as the system administrator shouldn't rely on any given numbers but the benchmarking process to make the system as secure as possible and reasonable in that regard.